R/DTW.R

```diff
@@ -12,10 +12,12 @@
 #'   evaluation grid for the first QTS.
 #' @param t2 An optional numeric vector of size \code{p2} specifying the
 #'   evaluation grid for the second QTS.
-#' @param distance_only A boolean specifyung whether to only compute distance
+#' @param distance_only A boolean specifyng whether to only compute distance
 #'   (no backtrack, faster). Default is \code{FALSE}.
-#' @param step_pattern The choice for the step pattern of the warping path. (default to \code{dtw::symmetric2},
-#' which is symmetric, normalizable with no local slope constraints. See \code{\link[dtw]{stepPattern}} for more information).
+#' @param step_pattern The choice for the step pattern of the warping path.
+#'   Defaults to \code{dtw::symmetric2}, which is symmetric,
+#'   normalizable with no local slope constraints.
+#'   See \code{\link[dtw]{stepPattern}} for more information.
```

Comment:
I have the impression that you indented by hand. Please use RStudio auto-indent option for comments.